drivers/hmis/app/controllers/ac_hmis/reports_controller.rb

```
@@ -0,0 +1,25 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module AcHmis
+  class ReportsController < Hmis::BaseController
+    def prevention_assessment_report
+      report = AcHmis::ReportApi.new.prevention_assessment_report(referral_id: params[:referral_id])
+
+      render body: report.body, status: report.http_status, content_type: 'application/pdf'
+    end
+
+    def consumer_summary_report
+      client = Hmis::Hud::Client.find_by(id: params[:client_id])
```

Let's raise an exception if the client is not found